volatility3/framework/plugins/windows/skeleton_key_check.py

```diff
@@ -202,21 +240,31 @@ def _find_and_parse_cryptdll(self, proc_list: Iterable) -> \
                                                                                  excp.layer_name))
                 continue
 
-            proc_layer = self.context.layers[proc_layer_name]
+            return proc, proc_layer_name
```

Comment:
I'd put this inside the try and lose the `continue`?  If the exception happens, the return won't, so it's the same outcome but fewer lines of code/flow to follow?  I'm ok if you want to keep it this way, but it just feels more confusing to me...

Reply:
changed.